packages/playground/src/style.css

```diff
@@ -155,3 +155,7 @@ body {
   padding: 10px;
   margin: 20px;
 }
+
+.output-dropdown{
+  float: right;
```

Comment:
nearly there, should be floating above the editor so it doesn't take vertical space. Look at using absolute position